src/schemas/parseJsonSchema.js

```diff
@@ -9,8 +9,7 @@ async function prepareJsonData(schema, uuid = null) {
     for (const [key, value] of Object.entries(schema)) {
         if (key === '_meta') {
             if (schema._meta && schema._meta.benchmark) {
-                // ~1MB payload record size
-                const size = (1 * 1012 * 1012) / 2;
+                const size = (schema._meta.benchmark) / 2;
```

Comment:
It still looks like the size is passed in through the json schema file, but ideally it would be a CLI option like the kafka-producer-perf-test `--record-size` so users could try different sizes without editing the json file

Reply:
Just refactored this! Initially, I misunderstood the suggestion! But it totally makes sense, now the `--record-size` or `-rs` for short, flag can be used with other schemas as well!

Also added tests.